uint16 serviceID
uint16 methodID
uint32 dataLength
uint16 clientID
uint16 sessionID
uint8 protocolVersion
uint8 interfaceVersion
uint8 messageType
uint8 returnCode
uint64 crc
uint32 length
uint32 sqc
uint32 dataID
uint32 timestampNanoseconds
uint32 timestampSeconds
uint8 timestampSyncStatus
uint32 eventDataQualifier
uint8 extendedQualifier
uint8 objectListNumOfObjects
objects[] objectArray